iiif-5-day-workshop/README.md

```diff
@@ -4,3 +4,5 @@
  (http://iiif.io).
  
  This course will introduce students to the basic concepts and technologies that make IIIF possible, allowing for guided, hands-on experience in installing servers and clients that support IIIF, and utilizing the advanced functionality that IIIF provides for interactive image-based research, such as annotation.
+
+[View this material in Gitbook form](https://iiif.github.io/training/iiif-5-day-workshop/)
```

Comment:
This will add the same info + link into the Gitbook...

Reply:
Maybe here instead? https://github.com/IIIF/training/blob/master/README.md

Reply:
META! 
But yeah, that's a probably a better spot. I'll rebase 👀